src/fbw/src/FlyByWireInterface.h

```diff
@@ -94,6 +97,26 @@ class FlyByWireInterface {
   bool last_ls1_active = false;
   bool last_ls2_active = false;
 
+  // Fix missing data for FDR Analysis
+  // controller input data
+  double inputElevator = 0.0;
+  double inputAileron = 0.0;
+  double inputRudder = 0.0;
+  // additional
+  double simulation_rate = 0.0;
+  // ambient data
+  double ice_structure_percent = 0.0;
+  double ambient_pressure_mbar = 0.0;
+  double ambient_wind_velocity_kn = 0.0;
+  double ambient_wind_direction_deg = 0.0;
+  double total_air_temperature_celsius = 0.0;
+  // aoa
+  double alpha_floor_condition = 0.0;
+  // these are not correct yet
+  double protection_ap_disc = 0.0;
+  double v_alpha_prot_kn = 0.0;
+  double v_alpha_max_kn = 0.0;
+
```

Comment:
Seems like these fields are not used and are duplicated from [AdditionalData.h](https://github.com/flybywiresim/a32nx/pull/7538/files#diff-6a74ba4307cc2097b8e85392beee956d961c74424e0d2192497afbc6c09bf88f)

Reply:
yea - my fault - interestingly these inits seem to not even be necessary. 
Will fix